plugins/pur/repository-proxy/pom.xml

```diff
@@ -56,8 +55,6 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
-      <version>${slf4j-api.version}</version>
-      <scope>provided</scope>
```

Comment:
Did you check if the jar continues not being placed in the pur plugin assembly / feature? Asking because you changed the scope of the dep.

Reply:
This is the osgi bundle made for dashboards-next, it doesn't affect the pdi-pur-plugin assembly. But yes, slf4j-api-1.7.7.jar is still there.